streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/util/CommonUtil.java

```diff
@@ -0,0 +1,14 @@
+package io.ray.streaming.runtime.util;
+
+import java.util.Map;
+
+/**
+ * Common tools.
+ */
+public class CommonUtil {
```

Comment:
Seems `Utils` is used more common in java

Reply:
We don't need to use plural unless the ```xxxUtil``` express multiple types using.
But for ```CommonUtil```, seems ```CommonUtils``` is better.